<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUiOpenedFilePicker</class>
 <widget class="QWidget" name="CUiOpenedFilePicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUiOpenedFilePicker</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLineEdit" name="m_filter">
     <property name="placeholderText">
      <string>Type to filter files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_btnUp">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../res/mac/res.qrc">
       <normaloff>:/res/icons/global-collapse.png</normaloff>:/res/icons/global-collapse.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="m_fileListView">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <property name="isWrapping" stdset="0">
      <bool>false</bool>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="uniformItemSizes">
      <bool>true</bool>
     </property>
     <property name="selectionRectVisible">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_btnDown">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../res/mac/res.qrc">
       <normaloff>:/res/icons/global-expand.png</normaloff>:/res/icons/global-expand.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../res/mac/res.qrc"/>
  <include location="../../res/mac/res.qrc"/>
 </resources>
 <connections/>
</ui>
